client/src/store/new-request-module.ts

```diff
@@ -2257,33 +2248,36 @@ export class NewRequestModule extends VuexModule {
           break
       }
 
-      const requestData: any = data && await this.addRequestActionComment(data)
-      try {
-        const response: AxiosResponse = requestData && await axios.post(`/namerequests`, requestData, {
-          headers: {
-            'Content-Type': 'application/json'
-          }
-        })
+      const requestData = data && await this.addRequestActionComment(data)
+      const response = requestData && await axios.post(`/namerequests`, requestData, {
+        headers: { 'Content-Type': 'application/json' }
```

Comment:
Just curious about the headers being defined here, in our other UI's we don't usually have to declare this afaik.
Is there reason we have to declare here in this ui?